components/constellation/constellation.rs

```diff
@@ -2083,9 +2059,12 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
             return ReadyToSave::NoRootFrame;
         }
 
-        // If there are pending loads, wait for those to complete.
-        if !self.pending_frames.is_empty() {
-            return ReadyToSave::PendingFrames;
+        if let Some(root_frame_id) = self.root_frame_id {
+            for frame in self.full_frame_tree_iter(root_frame_id) {
```

Comment:
Not sure of the performance consequences here when doing reftests. There shouldn't be many pipelines/frames.